README.md

```diff
@@ -30,7 +30,7 @@ virtualenv venv         # create the virtualenv in ./venv
 . venv/bin/activate    # activate it
 git clone https://github.com/mozilla-releng/mozilla-version
 cd mozilla-version
-pip install mozilla-version
+pip install --upgrade --require-hashes -r requirements/local.txt
```

Comment:
I could be wrong, but I think `--upgrade` won't do anything if the requirements file is all pinned anyway.

Reply:
Oh, you're right. Fixed!